usaspending_api/database_scripts/broker_matviews/broker_server.sql

```diff
@@ -1,26 +0,0 @@
-CREATE EXTENSION postgres_fdw;
```

Comment:
`fix_deleted_fabs.py` references this file in comments and error text.  Is `broker_server.sql` needed?  Do the comments need to be updated?  Can `fix_deleted_fabs.py` go away too?

Reply:
While it is a SQL file, the SQL isn't actually run by any scripts. I'll remove `fix_deleted_fabs.py` since it is old and hasn't been used recently